depthai_nodes/ml/parsers/utils/ppocr.py

```diff
@@ -0,0 +1,191 @@
+from typing import Tuple
```

Comment:
Would it make sense to rename this file to `ppdet` or something similar since this utils are not for OCR?

Reply:
Yes, that works as well